react.gradle

```diff
@@ -104,13 +104,12 @@ afterEvaluate {
 
         if (config.cliPath || config.nodeExecutableAndArgs) {
             if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                execCommand.addAll(["cmd", "/c", *nodeExecutableAndArgs, cliPath])
+                execCommand.addAll(["cmd", "/c", *nodeExecutableAndArgs, config.cliPath])
             } else {
-                execCommand.addAll([*nodeExecutableAndArgs, cliPath])
+                execCommand.addAll([*nodeExecutableAndArgs, config.cliPath])
             }
         } else {
-            throw new Exception("Missing cliPath or nodeExecutableAndArgs from build config. " +
-                "Please set project.ext.react.cliPath to the path of the react-native cli.js");
+            execCommand.add(defaultCliPath)
```

Comment:
shouldn't we distinguish between Windows and the rest here as well?

Reply:
Yeah, probably, we need a bit more work here. @Esemesek

Reply:
Don't we also need `nodeExecutableAndArgs` in here?

Reply:
I have rewrote the logic a bit to support both cases